jira/cli/templates.go

```diff
@@ -52,12 +52,18 @@ comments:
 `
 const default_edit_template = `# issue: {{ .key }}
 update:
+{{if .overrides.comment}}
   comment:
-    - add: 
+    - add:
         body: |~
           {{ or .overrides.comment "" | indent 10 }}
+{{end}}
+{{ if .overrides.label }}
```

Comment:
In theory the yamlFixup routine should strip out the empty `add: ` line, if it is not doing that there is a bug.